mods/ts/rules/shared-support.yaml

```diff
@@ -28,7 +28,7 @@ NAPULS:
 		Cost: 1000
 	Tooltip:
 		Name: EMP Cannon
-		Description: Disables vehicles. \nRequires power to operate.\n  Strong vs all ground units\n  Cannot target Aircraft
+		Description: Disables vehicles.\nRequires power to operate.\n  Strong vs Vehicles\n  Weak vs Infantry, Aircraft
```

Comment:
In the original it worked vs Cyborgs and Buildings, too, but it currently doesn't in OpenRA, so this description is ok for now.